### ISSUE TITLE
[Adapter Cards-Schema Explorer]: Narrator is announcing some extra irrelevant information with the "OK button" info.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7886]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:
OS: Version 22H2 (OS Build 25197.1000)
Edge Dev: version 107.0.1379.1 (Official build) dev (64-bit)
Screen Reader: Narrator
Application URL: [[[https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)]([https://adaptivecards.io/explorer/)](https://adaptivecards.io/%5D(https://adaptivecards.io/explorer/)%5D(https://adaptivecards.io/))

Pre-Requisites:
Run the Narrator using "Control+ Windows +Enter" short key in the scan mode.

Repro Steps:
1.Open above URL with V- credential.
2.Navigate to 'Schema Explorer' link using tab key and press enter key to activate it.
3.Navigate to 'Container" link present under "Containers" from the left navigation using the tab key and activate it using Enter key.
4. Navigate to the adaptive cards present under it using the Tab key.
5. Navigate to the "Submit Action" button with tab key using narrator and activate using Enter key.
6. Observe the issue here.

Actual Result:
Narrator is announcing like "Adaptivecards.io says OK button 1of1".

Expected Result:
Narrator shouldn't give the position of the OK button as there is only one button in the comment box, and it should announce it like "Adaptivecards.io says OK button".

Oservation:
Issue exists throughtout the application wherever the "OK" button is present after submiting the adaptive cards.

User Experience:
Screen reader users will be impacted, and they will get confusion over here if narrator announces the position of the OK button.

MAS Reference:
https://liquid.microsoft.com/Web/Object/Read/ms.accessibility/Requirements/01.03.01

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

Uploading narrator announces the position of OK button which is not applicable.mp4…

### Card JSON

```JSON
NA
```

### Sample Code Language

NA

### Sample Code

NA

**Comment by vagpt** [comment]
This issue is not repro, hence closing this issue.
